File: El Mamoun/Lv 1/Problem 39.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Problem 39: Write a program to read a TotalBill and CashPaid and calculate the remainder to be paid back.
*/


namespace Problem_39
{
    internal class Program
    {
        static float ReadNumber(string Message)
        {

            float Number = 0;
            Console.WriteLine(Message);
            Number = Convert.ToSingle(Console.ReadLine());

            return Number;
        }

        static float CalPayRemainder(float TotalBill,float CashPaid)
        {
            return (CashPaid - TotalBill);
        }

        static void Main(string[] args)
        {
            float TotalBill = ReadNumber("Please Enter a TotalBill");
            float CashPaid = ReadNumber("Please Enter a CashPaid");

            Console.WriteLine(CalPayRemainder(ReadNumber("Please Enter a TotalBill"), ReadNumber("Please Enter a CashPaid")));
            Console.WriteLine(CalPayRemainder(TotalBill, CashPaid));

            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/problem 39.cpp
#include<iostream>
using namespace std;

/*
Problem 39: Write a program to read a TotalBill and CashPaid and calculate the remainder to be paid back.
*/

int ReadPostiveNumber(string message)
{
	int number{ 0 };

	do
	{
		cout << message;
		cin >> number;

	} while (number < 0);

	return number;
}

float CalculateRemainder(float TotalBill, float CashPaid)
{
	return (CashPaid - TotalBill);
}


int main()
{

	float TotalBill = ReadPostiveNumber("Please Enter a Total Bill? ");
	float CashPaid  = ReadPostiveNumber("Please Enter a Cash Piad? ");

	cout <<"Result = "<< CalculateRemainder(TotalBill, CashPaid);

	return 0;
}